cg/models/rnafusion/rnafusion.py

```diff
@@ -93,3 +100,29 @@ class RnafusionAnalysis(AnalysisModel):
     """
 
     sample_metrics: Dict[str, RnafusionQCMetrics]
+
+
+class RnafusionDeliverables(PipelineDeliverables):
+    """Specification for pipeline deliverables."""
+
+    @staticmethod
+    def get_deliverables_template() -> List[dict]:
+        """Return deliverables file template content."""
+        return ReadFile.get_content_from_file(
+            file_format=FileFormat.YAML,
+            file_path=resources.RNAFUSION_BUNDLE_FILENAMES_PATH,
+        )
+
+    @classmethod
+    def get_deliverables_for_case(cls, case_id: str, case_path: Path):
```

Comment:
add return type hint

Reply:
Maybe `case_id` and `case_path` can be the input of the `RnafusionDeliverables` class

Reply:
I cannot add the return type hint as the type hint is the class itself

Reply:
I wonder if that is not a good sign that refactoring is needed.

Reply:
Alright, this class got a bit of criticism so I removed it in favour of the more general class PipelineDeliverables; and  moved Rnafusion specific methods to workflow instead of as part of the base model

Reply:
I like the current solution 💯